src/main/java/io/github/wistefan/dcql/ClaimsEvaluator.java

```diff
@@ -0,0 +1,220 @@
+package io.github.wistefan.dcql;
+
+import io.github.wistefan.dcql.model.ClaimsQuery;
+import io.github.wistefan.dcql.model.credential.*;
+import lombok.extern.slf4j.Slf4j;
+
+import java.util.*;
+import java.util.stream.Collectors;
+
+@Slf4j
+public class ClaimsEvaluator {
+
+	private static final String SD_KEY = "_sd";
```

Comment:
Maybe a additional documentation for this key would be useful. At least I was unfamiliar with the selective disclosure field specifics